pkg/fleet/installer/packages/apm_library_dotnet_windows.go

```diff
@@ -50,7 +50,11 @@ func SetupAPMLibraryDotnet(ctx context.Context, _ string) (err error) {
 		return err
 	}
 	dotnetExec := exec.NewDotnetLibraryExec(getExecutablePath(installDir))
-	_, err = dotnetExec.Install(ctx, getLibraryPath(installDir))
+	_, err = dotnetExec.InstallVersion(ctx, getLibraryPath(installDir))
+	if err != nil {
+		return err
+	}
+	_, err = dotnetExec.EnableIISInstrumentation(ctx, getLibraryPath(installDir))
```

Comment:
Is there a benefit to calling `getLibraryPath(installDir)` twice here over saving to a variable?